framework/elastic_query_builder.rb

```diff
@@ -143,7 +143,12 @@ def construct_es_query_term filter_key, value
       {
         multi_match: { query: value, type: flag, fields: multi_match_fields }.compact
       }
-    when "term", "fuzzy", "prefix", "wildcard", "regexp"
+    when "fuzzy"
+      multi_match_fields = fields == ["_all"] ? nil : fields
+      {
+        multi_match: { query: value, fields: multi_match_fields, fuzziness: "AUTO" }.compact
```

Comment:
I think the compact can be dropped if we implement the above suggestion

Reply:
perhaps better to keep it, if it's allowed to have fields unspecified in the api (I'm unsure)